app/uk/gov/hmrc/helptosavefrontend/controllers/ReminderController.scala

```diff
@@ -164,8 +164,8 @@ class ReminderController @Inject() (
           },
           success =>
             htsContext.userDetails match {
-              case Left(missingUserInfos) =>
-                logger.warn(s"Email was verified but missing some user info $missingUserInfos")
```

Comment:
❓ These missingInfo logs sound like logging an error rather than user data - if we're logging "missing fields" I assume these are the fields that the user did not fill, so we probably can log it.

Reply:
good spot!